### ISSUE TITLE
Exception in callback of async function: TypeError: Cannot read property 'length' of undefined

### ISSUE THREAD

**Issue by domgolonka** [MiroHibler/meteor-preloader#19]
```
Exception in callback of async function: TypeError: Cannot read property 'length' of undefined
    at Object.Preloader.loadLibs (http://localhost:3000/packages/miro_preloader.js?07b361b1abe7bd3b9526e0031b650b48d81b0173:525:34)
    at RouteController.extend.waitOn (http://localhost:3000/packages/miro_preloader.js?07b361b1abe7bd3b9526e0031b650b48d81b0173:585:21)
    at http://localhost:3000/packages/iron_router.js?a427868585af16bb88b7c9996b2449aebb8dbf51:438:22
    at Array.forEach (native)
    at Function._.each._.forEach (http://localhost:3000/packages/underscore.js?0a80a8623e1b40b5df5a05582f288ddd586eaa18:156:11)
    at RouteController._runRoute (http://localhost:3000/packages/iron_router.js?a427868585af16bb88b7c9996b2449aebb8dbf51:437:5)
    at Function.Route.dispatch (http://localhost:3000/packages/iron_router.js?a427868585af16bb88b7c9996b2449aebb8dbf51:844:18)
    at route (http://localhost:3000/packages/iron_router.js?a427868585af16bb88b7c9996b2449aebb8dbf51:710:11)
    at boundNext (http://localhost:3000/packages/iron_middleware-stack.js?0e0f6983a838a6516556b08e62894f89720e2c44:424:31)
    at Meteor.bindEnvironment (http://localhost:3000/packages/meteor.js?43b7958c1598803e94014f27f5f622b0bddc0aaf:983:22)
```

Here is my router:

```
SettingsController = PreloadController.extend({
    'preload': {
        'async': [
            '/plugins/jquery-scrollbar/jquery.scrollbar.min.js',
            '/plugins/pace/pace.min.js',
            '/plugins/jquery-numberAnimate/jquery.animateNumbers.js'
        ],
    }
});

Router.route('settings', function(){
    Router.go('settingsIndex');
});
Router.route('settings/index/:limit?/', {
    name: 'settingsIndex',
    controller: SettingsController,
    action: 'index',
});
```

**Comment by domgolonka** [comment]
I also tried this but it gives me the same error.
```
SettingsController = PreloadController.extend({
    'preload': {
    }
});

Router.route('settings', function(){
    Router.go('settingsIndex');
});
Router.route('settings/index/:limit?/', {
    name: 'settingsIndex',
    controller: SettingsController,
    action: 'index',
});
```

**Comment by leebenson** [comment]
The problem seems to be that `preload` requires a full object, like:

```
preload: {
    styles: ['/css/blank.css'],
    async: [
        'https://s3-eu-west-1.amazonaws.com/path/to/file.js'
    ]
}
```

Even if you don't need 'styles' in the above example, you need to set it to _something_ or you get the error. This seems to have been introduced in v1.2.1

**Comment by MiroHibler** [comment]
Fixed with v1.2.3.

Thanks guys! :)
